CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 ### Fixed 🐞
 
 -   Fix SonarImporter requesting no metrics from SonarQube when the list of metrics was left empty [#2913](https://github.com/MaibornWolff/codecharta/pull/2913)
+-   Change CompressionStreamHandler usage of Java11 Method to Java9 [#2930](https://github.com/MaibornWolff/codecharta/pull/2930)
```

Comment:
I would name the affected Method in the Changelog, so that it is easier to track this error.

Reply:
Addressed in d35f7cb 👍